src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -211,11 +219,47 @@ void onTerminate(@Nullable Module module,
                        @SuppressWarnings("UnusedParameters") @NotNull String... args) {
         ApplicationManager.getApplication().invokeLater(() -> {
           if (workingDir != null && module != null && !module.isDisposed()) {
+            final Project project = module.getProject();
+
             // Enable Dart.
             FlutterSdkUtil.enableDartSupport(module);
-            final FileEditorManager manager = FileEditorManager.getInstance(module.getProject());
-            // Open main.
-            final VirtualFile main = workingDir.findFileByRelativePath("lib/main.dart");
+            final FileEditorManager manager = FileEditorManager.getInstance(project);
+            
+            // Create a basic run configuration.
```

Comment:
I don't think `flutter create` is the way to go here since I think run configs live in project metadata somewhere?

As for a pre-existing run config, I guess I assumed that with a fresh project there couldn't be one but since this could be called during module creation maybe that's not safe.

https://github.com/flutter/flutter-intellij/issues/245